modules/monitoring-disabling-the-local-alertmanager.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_the_local_alertmanager_{context}"]
```

Comment:
```suggestion
[id="monitoring-disabling-the-local-alertmanager_{context}"]
```
This should match the filename exactly: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata and should use dashes instead of underscores

Reply:
Fixed.